### PR TITLE
Merging VLMCSD database and License Manager database

### PR DESCRIPTION
**Pull request by shiroinekotfs**
## What's new in this PR?

This PR merges the issue, raised in #36 by @herokukms. The merge resolves these existing, and look-over problems with both VLMCSD and License Manager

1. Export database from License Manager (with `.xml` format) to native VLMCSD database (with `.kmd` format) requires the removal of SQL Server, SCCM, and Visual Studio keys. This process is now unnecessary because the export process only applies to defined, known **Build Number**, **LCID**, and **KMSItemID**, while these keys are no longer inside the normal database. However, it is still can be seen in **Store license keys**

<p align="center">
<image  src="https://github.com/TheFlightSims/windowsserver-mgmttools/assets/115929530/f874a008-7302-413a-80b0-604098164d41" />
</p>

2. Restructured XML database. That means, only the License Manager bundled with can be compatible with this License Manager. Other License Manager versions (including released and will-be-released versions) can still import this XML database but may raise an error. Please proceed with caution and contact @TheFlightSims as soon as possible when the error happens.

3. Removal of `license-manager` folder in `database-config`. As I mentioned, the License Manager XML database is already bundled within the application, so separating it is not needed.

4. Removal of `release` build mode in VLMCSD on Visual Studio. (Why do we need it, when we have `Release` and `Debug`?)

Other changes are not mentioned, because these are minor fixes, to improve and make the application more secure.

## Suggestion and Report problems

If you have any questions, you can try to ask here, on the [issue page](https://github.com/TheFlightSims/windowsserver-mgmttools/issues), or DM any assigned developers to this PR via their GitHub profile.

Once again, sorry for the inconvenience we have made for you.

### CODE REVIEW

**Comment by TheFlightSimsOfficial** [comment]
> @TheFlightSimsOfficial úm ba la, mau hiện lên =))

Hi @shiroinekotfs, long time no see you speaking Vietnamese publicly? LOL

File: vlmcsd-beta/vlmcsd/src/kms.c [hunk 1]
```diff
@@ -74,14 +74,27 @@ const char* getNextString(const char* s)
 
 #ifndef NO_RANDOM_EPID
 static const uint16_t LcidList[] = {
```
Comment (TheFlightSimsOfficial) [inline vlmcsd-beta/vlmcsd/src/kms.c:76]: It's not needed.

File: vlmcsd-beta/license-manager/ProductKeys.xaml.cs [hunk 2]
```diff
@@ -17,16 +17,16 @@ public partial class ProductKeys
...
-            new ProductKey ("Windows Preview Core Country Specific", "NDTWC-J28B6-RTR3C-CPRHW-2GV83", KeyType.Gvlk),
-            new ProductKey ("Windows Preview Professional N", "G2DR7-7N9C8-KBK9V-D4W3M-H49R8", KeyType.Gvlk),
-            new ProductKey ("Windows Preview Enterprise N", "N8BR7-D8XB9-939VB-FMFWD-KWXR9", KeyType.Gvlk),
-            new ProductKey ("Windows Preview Standard Server", "YNBF9-GPVTG-FFHQC-MJR4B-B4CQX", KeyType.Gvlk),
-            new ProductKey ("Windows Preview Multipoint Server Standard", "3CK7G-VRNKC-8QBFR-9G8HC-YKG8W", KeyType.Gvlk),
-            new ProductKey ("Windows Preview Multipoint Server Premium", "GRMDJ-JNF7H-W9WTC-WKQRG-H8KW6", KeyType.Gvlk),
-            new ProductKey ("Windows 10 Professional", "VK7JG-NPHTM-C97JM-9MPGT-3V66T", KeyType.StoreLicense ),
-            new ProductKey ("Windows 10 Home", "YTMG3-N6DKC-DKB77-7M9GH-8HVX7", KeyType.StoreLicense ),
-    };
+            new ProductKey ("Windows 10/11 Professional", "VK7JG-NPHTM-C97JM-9MPGT-3V66T", KeyType.StoreLicense ),
```
Comment (TheFlightSimsOfficial) [inline vlmcsd-beta/license-manager/ProductKeys.xaml.cs:20]: Not clever at all. You removed Preview Keys. It's very essential for future researcher/developer to know on deployment old Windows versions.

File: vlmcsd-beta/license-manager/LmInternalDatabase.xml [hunk 3]
```diff
@@ -643,63 +520,98 @@
...
-
 			<KmsItem DisplayName="Windows Preview" Id="5f94a0bb-d5a0-4081-a685-5819418b2fe0" IsPreview="false" DefaultKmsProtocol="5.0">
 				<SkuItem DisplayName="Windows 10 Enterprise Preview" Id="cde952c7-2f96-4d9d-8f2b-2d349f64fc51" Gvlk="2MP7K-98NK8-WPVF3-Q2WDG-VMD98" />
 				<SkuItem DisplayName="Windows 10 Professional Preview" Id="a4383e6b-dada-423d-a43d-f25678429676" Gvlk="MTWNQ-CKDHJ-3HXW9-Q2PFX-WB2HQ" />
 				<SkuItem DisplayName="Windows 10 Professional WMC Preview" Id="cf59a07b-1a2a-4be0-bfe0-423b5823e663" Gvlk="MY4N9-TGH34-4X4VY-8FG2T-RRDPV" />
 				<SkuItem DisplayName="Windows 8.x Preview" Id="2b9c337f-7a1d-4271-90a3-c6855a2b8a1c" Gvlk="MPWP3-DXNP9-BRD79-W8WFP-3YFJ6" />
 				<SkuItem DisplayName="Windows 8.x Preview ARM" Id="631ead72-a8ab-4df8-bbdf-372029989bdd" />
 			</KmsItem>
-
+			<KmsItem DisplayName="Windows 10 2019 (Volume)" Id="11b15659-e603-4cf1-9c1f-f0ec01b81888" IsPreview="false" CanMapToDefaultCsvlk="false" DefaultKmsProtocol="6.0">
```
Comment (TheFlightSimsOfficial) [inline vlmcsd-beta/license-manager/LmInternalDatabase.xml:530]: Sorting?

**Review by TheFlightSimsOfficial — COMMENTED** [review]
You need to restore old product keys you have deleted.

File: vlmcsd-beta/vlmcsd/src/kms.c (hunk 1 above)
Comment (shiroinekotfs) [reply vlmcsd-beta/vlmcsd/src/kms.c:76]: Well, it's needed for sorting Local ID (LCID)

File: vlmcsd-beta/license-manager/ProductKeys.xaml.cs (hunk 2 above)
Comment (shiroinekotfs) [reply vlmcsd-beta/license-manager/ProductKeys.xaml.cs:20]: Well, I'll fix it.

File: vlmcsd-beta/license-manager/LmInternalDatabase.xml (hunk 3 above)
Comment (shiroinekotfs) [reply vlmcsd-beta/license-manager/LmInternalDatabase.xml:530]: Yes.

**Review by TheFlightSimsOfficial — APPROVED** [review]
Now it seems legit. Hope this patch will release soon.

**Pull request merged**